File: app/src/main/java/com/mvrborges/projectgold/configs/ForgetPassword.java
package com.mvrborges.projectgold.configs;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import com.mvrborges.projectgold.R;

public class ForgetPassword extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        this.setTitle(getResources().getText(R.string.forgetPassword_title));
        setContentView(R.layout.activity_forget_password);


    }

    public void onClickBtnCancel(View view) {
        Intent inCreate = new Intent(ForgetPassword.this, Login.class);
        startActivity(inCreate);
    }

    public void onClickBtnCreate(View view) {
        Intent inCreate = new Intent(ForgetPassword.this, Login.class);
        startActivity(inCreate);
    }
}

File: app/src/main/java/com/mvrborges/projectgold/configs/ForgetPassword.kt
package com.mvrborges.projectgold.configs

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View

import com.mvrborges.projectgold.R

class ForgetPassword : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        this.title = resources.getText(R.string.forgetPassword_title)
        setContentView(R.layout.activity_forget_password)


    }

    fun onClickBtnCancel(view: View) {
        val inCreate = Intent(this@ForgetPassword, Login::class.java)
        startActivity(inCreate)
    }

    fun onClickBtnCreate(view: View) {
        val inCreate = Intent(this@ForgetPassword, Login::class.java)
        startActivity(inCreate)
    }
}
